orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -59,7 +59,14 @@ class CreateWebhookTask implements RetryableTask {
     def response
     try {
       response = webhookService.exchange(stageData.method, stageData.url, stageData.payload, stageData.customHeaders)
-    } catch (HttpStatusCodeException e) {
+    } catch(IllegalArgumentException e) {
+      if (e.cause instanceof UnknownHostException) {
+        String errorMessage = "name resolution failure in webhook for pipeline ${stage.execution.id} to ${stageData.url}, will retry."
```

Comment:
If this error is going to be user facing I'd suggest stripping the `stage.execution.id` from it, that's confusing.

could just tack it onto the log ... `log.warn("${errorMessage} (executionId: ${stage.execution.id}", e)`

Reply:
Tough to say whether logging the entire stack trace is going to be super noisy and whether we only really need `e.message` for an `UnknownHostException`

🤷‍♂️

Reply:
Thoughts about the same treatment for the `MonitorWebhookTask`?

In theory it's equally susceptible.

Reply:
I just came back to add a note about `MonitorWebhookTask` but you're already on, thanks @ajordens! :)

Reply:
@ajordens I don't think it's supposed to be user facing, if it does persist the task would fail with a timeout I think

Good catch about the `MonitorWebhookTask`, will add it